cfme/storage/volume.py

```diff
@@ -297,6 +294,9 @@ class VolumeCollection(BaseCollection):
     """Collection object for the :py:class:'cfme.storage.volume.Volume'. """
     ENTITY = Volume
 
+    # Navigation menu option
+    nav = ['Storage', 'Block Storage', 'Volumes']
```

Comment:
**REQUIRED:** As nice as this is, right now no one else is doing it, we can propose it somewhere but I don't want to include it in this PR please

Reply:
Looking again I see that this has been in for a while, I would ask you to please remove it as part of this PR till we decide to do it as a standard, Thanks

Reply:
Ok @psav going to change in our standard format. Thanks